ui/lib/pki/addon/components/pki-tidy-form.ts

```diff
@@ -0,0 +1,65 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import Component from '@glimmer/component';
+import errorMessage from 'vault/utils/error-message';
+import { action } from '@ember/object';
+import { inject as service } from '@ember/service';
+import { task } from 'ember-concurrency';
+import { waitFor } from '@ember/test-waiters';
+import { tracked } from '@glimmer/tracking';
+
+import type RouterService from '@ember/routing/router-service';
+import type PkiTidyModel from 'vault/models/pki/tidy';
+import type { FormField, TtlEvent } from 'vault/app-types';
+
+interface Args {
+  tidy: PkiTidyModel;
+  tidyType: string;
+  onSave: CallableFunction;
+  onCancel: CallableFunction;
+}
+
+interface PkiTidyTtls {
+  intervalDuration: string;
+  acmeAccountSafetyBuffer: string;
+}
+interface PkiTidyBooleans {
+  enabled: boolean;
+  tidyAcme: boolean;
+}
+
+export default class PkiTidyForm extends Component<Args> {
+  @service declare readonly router: RouterService;
+
+  @tracked errorBanner = '';
+  @tracked invalidFormAlert = '';
+
+  get intervalDurationAttr() {
```

Comment:
did this because with the model refactor `allFields` was removed from the model

Reply:
There is now `allByKey` on the model so you can do this same thing like `this.args.tidy.allByKey.intervalDuration` instead

Reply:
amazing! that was showing up `undefined` in some places, but maybe I had something wrong 🤔 - thanks for pointing that out!